tox.ini

```diff
@@ -58,7 +58,7 @@ commands =
 description = Run unit tests
 deps =
     Jinja2==3.1.2
-    mysql-connector-python==8.0.30
```

Comment:
should we replicate this change to the `integration-database` tox target?

Reply:
Actually, should I simply unpin it, like in `integration-database` tox target?
I didn't notice it there. Tnx!

Reply:
+1 to unpin - one less thing to control for and normally quick to spot on failure

Reply:
Done.